command/commands/team.py

```diff
@@ -7,7 +7,31 @@ class TeamCommand:
     """Represent Team Command Parser."""
 
     command_name = "team"
-    help = ""
+    help = """# Team Command Reference
+    `@rocket team`
+    All parameters with whitespace must be enclosed by quotation marks.
+    ## Options to specify input
+    * `list`
+        * outputs the Github team names and display names of all teams
+    * `view` GITHUB_TEAM_NAME
+        * view information and members of a team
+    * `help`
+        * outputs options for team commands
+    ## TEAM LEAD or ADMIN only
+    * `create` GITHUB_TEAM_NAME [`--name` DISPLAY_NAME] [`--platform` PLATFORM]
+        * create a new team with a Github team name and optional parameters
+        * the user will be automatically added to the new team
+    The following can only be used by a team lead in the team or an admin:
+    * `edit` GITHUB_TEAM_NAME
+        * [`--name` DISPLAY_NAME]
+        * [`--platform` PLATFORM]
+            * edit properties of specified team
+    * `add` GITHUB_TEAM_NAME @Slack User
+        * add the specified user to the team
+    * `remove` GITHUB_TEAM_NAME @Slack User
+        * remove the specified user from the team
+    * `delete` GITHUB_TEAM_NAME
+        * permanently delete the specified team"""
```

Comment:
Please un-indent the string; the indents would carry over, making the string that gets passed to slack a bit padded (and even more importantly, ugly).

Reply:
Would a better way be to read the `docs/TeamCommand.md` file and just dump it into the help string? A bit of a dilemma here, that you would have to choose between modifying both strings when the docs change, or reading and copying the text of a docs file.

Reply:
I was trying to grab the text from the file, but I was confused at what the path should be. 
`help = open("docs/TeamCommand.md", "r").read()` works when you run pytest in the rocket2.0 folder, but not when you run pytest in the test folder, for example.

Reply:
Good point